tests/test_upload_geometries.py

```diff
@@ -11,27 +12,29 @@
 
 
 def test__merge_feature_collection():
-    fc = {
-        "type": "FeatureCollection",
-        "features": [
-            {
-                "type": "Feature",
-                "geometry": {
-                    "type": "Polygon",
-                    "coordinates": [
-                        [
-                            [-20.0, 10.0],
-                            [-10.0, 10.0],
-                            [-10.0, 0.0],
-                            [-20.0, 0.0],
-                            [-20.0, 10.0],
-                        ]
-                    ],
-                },
-                "properties": {"name": "/vsis3/gfw-data-lake-test/10N_020W.tif"},
-            }
-        ],
-    }
+    fc = FeatureCollection(
+        **{
+            "type": "FeatureCollection",
+            "features": [
+                {
+                    "type": "Feature",
+                    "geometry": {
+                        "type": "Polygon",
+                        "coordinates": [
+                            [
+                                [-20.0, 10.0],
+                                [-10.0, 10.0],
+                                [-10.0, 0.0],
+                                [-20.0, 0.0],
+                                [-20.0, 10.0],
+                            ]
+                        ],
+                    },
+                    "properties": {"name": "/vsis3/gfw-data-lake-test/10N_020W.tif"},
+                }
+            ],
+        }
+    )
```

Comment:
Do we still need this test? I believe we substituted it by fetching the metadata early on.

Reply:
There should be a test instead which checks that existing tiles and current tiles are correctly merged into one geojson

Reply:
Yup. Done.